test/synthetics/latest_release_cdn.js

```diff
@@ -0,0 +1,144 @@
+ * 
+ * How to install:
+ *   1. Go to Synthetics > Create monitor in NR1.
+ *   2. Select Scripted API check.
+ *   3. Define appropiate settings and copy/paste this script as last step.
+ *   4. Optionally, create an Alert Condition to get notifications.
+ */
+var assert = require('assert');
+var downloadServerBaseURI = `https://download.newrelic.com/infrastructure_agent`;
+var githubOrg = `https://api.github.com/repos/newrelic`;
+var allRepos = ['infrastructure-agent','nri-jmx', 'nrjmx', 'nri-snmp'];
+/* 
+ * Other OHIs: to consider in a separate synthetic check and alert
+'nri-mysql', 'nri-apache', 'nri-cassandra', 'nri-consul', 'nri-couchbase', 
+'nri-elasticsearch', 'nri-f5', 'nri-haproxy', 'nri-kafka', 'nri-memcached', 
+'nri-mongodb', 'nri-mssql', 'nri-mysql', 'nri-nagios', 'nri-nginx', 'nri-oracledb', 
+'nri-postgresql', 'nri-rabbitmq', 'nri-redis'];
+*/
+var arm64Unsupported = ['nri-oracledb'];
+var windowsUnsupported = ['nri-oracledb'];
+
+function checkLatestReleases(repos){
+  repos.forEach(repo => {
+      console.log(`Getting ${githubOrg}/${repo}/releases/latest`);
+      $http.get({
+        'uri': `${githubOrg}/${repo}/releases/latest`,
+        'headers': {'User-Agent': 'newrelic-ohai'}
+      },
+        function (err, response, body) {
+          assert.equal(response.statusCode, 200, `Expected 200 getting releases got ${response.statusCode} ${err}`);
+          
+          // Our release tags might or not contain a 'v' prefix (1.0.0 vs v1.0.0)
+          var latestRelease = JSON.parse(body).tag_name.replace('v','');
+          checkPackagesInCDN(repo, latestRelease);
+      }
+    );
+  });
+}
+
+function checkPackagesInCDN(repo, latestRelease){
+    // need to consider arm64 vs aarch64 vs noarch alternatives
+    var archs = [
+     {
+       archInUrl: 'x86_64',
+       archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+    }];
+    if (arm64Unsupported.indexOf(repo) < 0){
+      archs.push({ 
+        archInUrl: 'x86_64', 
+        archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+      });
+    } 
+
+    // except from infra-agent, all OHIs have consistent repos and package names.
+    var packagePrefix = repo === 'infrastructure-agent' ? 'newrelic-infra' : repo;
+
+    // TODO: add checks for all OS versions (blocked until all OHIs are released) 
+    // TODO: add DockerHub images checks
+    checkSLES(packagePrefix, latestRelease, archs);
+    checkAL2(packagePrefix, latestRelease, archs);
+    checkRHEL(packagePrefix, latestRelease, archs);
+    checkDeb(packagePrefix, latestRelease, archs);
+
+    if (windowsUnsupported.indexOf(packagePrefix) < 0){
+      checkWindows(packagePrefix, latestRelease);
+    }
+}
+
+function checkSLES(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.sles12.4.${archs[i].archInPackage}.rpm` : 
```

Comment:
We are not checking all possible versions of SLES, should we?

Reply:
I'll add all versions on all OSs in the next iteration (initially this was blocked / related to having all OHIs in all OS versions published - I think this is solved now).